auto_remediate/config_rules.py

```diff
@@ -7,6 +7,29 @@ class ConfigRules:
     def __init__(self, logging):
         self.logging = logging
 
+        self._client_rds = None
+        self._client_s3 = None
+
+    @property
+    def client_rds(self):
+        if not self._client_rds:
+            self._client_rds = boto3.client("rds")
+        return self._client_rds
+
+    @client_rds.setter
```

Comment:
I realized that we actually don't need `@property.setters` anymore, we can get rid of these and keep the `@property` for lazy fetching

Reply:
890c7b4